### PR TITLE
make kubectl --raw consistent for create, update, get, delete

### PR DESCRIPTION
**Pull request by deads2k**
We have `kubectl create --raw` and `kubectl get --raw`.  This rounds out the featureset with `kubectl replace --raw` and `kubectl delete --raw`.

```release-note
add  `kubectl replace --raw` and `kubectl delete --raw` to have parity with create and get
```

@kubernetes/sig-cli-maintainers 
/kind cleanup
/priority important-longterm

### CODE REVIEW

File: pkg/kubectl/cmd/create/create.go [hunk 1]
```diff
@@ -20,10 +20,11 @@ import (
 	"fmt"
 	"io"
 	"net/url"
-	"os"
 	"runtime"
 	"strings"
 
+	"k8s.io/kubernetes/pkg/kubectl/cmd/rawhttp"
```
Comment (liggitt) [inline pkg/kubectl/cmd/create/create.go]: nit: group with other k8s imports (applies to all)

File: pkg/kubectl/cmd/rawhttp/raw.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
+package rawhttp
```
Comment (liggitt) [inline pkg/kubectl/cmd/rawhttp/raw.go]: might as well start this package under `k8s.io/kubectl`

**Review by liggitt — COMMENTED** [review]
nits, lgtm

File: pkg/kubectl/cmd/rawhttp/raw.go [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+	switch requestType {
+	case "GET":
+		request = restClient.Get().RequestURI(url)
+	case "PUT":
+		request = restClient.Put().RequestURI(url).Body(data)
+	case "POST":
+		request = restClient.Post().RequestURI(url).Body(data)
+	case "DELETE":
+		request = restClient.Post().RequestURI(url).Body(data)
+	}
```
Comment (liggitt) [inline pkg/kubectl/cmd/rawhttp/raw.go]: default case that errors?

**Comment by deads2k** [comment]
minor fix up for stock flags being added.  retagging.

**Pull request merged**